minigui/serve.py

```diff
@@ -44,9 +44,14 @@
     "--engine",
     default="py",
     type=str,
-    help="Which Minigo engine to use: \"py\" for the Python implementation, "
-         "\"cc\" for the C++ implementation.")
+    help="Which Minigo engine to use: \"py\" for the Python engine, or "
```

Comment:
Is it worth putting a validator on this?

Reply:
Oof, this help text was pretty wrong.

The problem is that the supported list of engines changes depending on how the C++ binary was compiled. I've changed text to describe how to find out the actually supported list.